### DISCUSSION TITLE
Different Icons for different outputs

### DISCUSSION THREAD

**Discussion by Red-AM** [File-New-Project/EarTrumpet#1011] [Ideas]
It would be cool if you could set specific icons for specific outputs. For example, if my headphones are the current output it would show a headphone icon. Being able to use your own icons would be cool too, but just having the option and a bunch of different default icons to choose from would also work. 

The reason I'd want this feature is so that I can just do a quick glance at my taskbar to know what output I'm currently on without having to move my cursor to it.

**Comment by GR8Dane604** [comment]
That would be awesome. You can customize the icon for the audio device, so ideally EarTrumpet would use that one since it's already defined in Windows.

**Comment by ghost** [comment]
Why still not implemented?

**Reply by riverar** [reply]
@direrFlood Development is slow as I'm the only maintainer at this time and have a lot of higher priority items to work on.

**Comment by oxysoft** [comment]
Also want to share my love for this, I switch audio device constantly and it's hard to keep track of! As a game developer by trade, I eat visual telegraphing for breakfast.

Some ideas for the icon...

1. A whole pair of headphones and position the waves between the two ears.
2. One ear on the side like the speaker, perhaps with a short tick extending and disappearing to imply the headband.
